src/StrList.cc

```diff
@@ -49,6 +49,21 @@ strListIsMember(const String * list, const SBuf &m, char del)
     return 0;
 }
 
+bool
+strListIsMember_if(const String *list, char del, MemberFilter filter)
```

Comment:
Use const if possible:
```suggestion
strListIsMember_if(const String *list, const char del, const MemberFilter filter)
```

Reply:
commit 099867f3cf